Sprint-2/implement/contains.test.js

```diff
@@ -33,3 +33,37 @@ test.todo("contains on empty object returns false");
 // Given invalid parameters like an array
 // When passed to contains
 // Then it should return false or throw an error
+
+
+
+describe("contains function", () => {
+  // Test: empty object
+  test("should return false when passed an empty object", () => {
+    expect(contains({}, "a")).toBe(false);
+  });
+
+  // Test: object with properties and existing key
+  test("should return true when the object contains the specified property", () => {
+    expect(contains({ a: 1, b: 2 }, "a")).toBe(true);
+  });
+
+  // Test: object with properties and non-existing key
+  test("should return false when the object does not contain the specified property", () => {
+    expect(contains({ a: 1, b: 2 }, "c")).toBe(false);
+  });
+
+  // Test: invalid parameters (array as first argument)
+  test("should throw an error when the first argument is an array", () => {
+    expect(() => contains([1, 2, 3], "1")).toThrow("Invalid input: first argument must be an object");
```

Comment:
can you explain better i would undrstnad better.

Reply:
My comment was about `typeof [1, 2, 3] === "object"` is true. So it make sense for
`contains([1, 2, 3], "1")` to return true because "0", "1", "2"  are the keys to that array.

I didn't notice the spec mentions "Given invalid parameters like an array", so don't worry about this.

Reply:
expect(contains({ a: 1, b: 2 }, "a")).toBe(true); but there is not key there are just numbers